config/protocol.privnet.yml

```diff
@@ -51,6 +51,11 @@ ApplicationConfiguration:
     Enabled: true
     EnableCORSWorkaround: false
     Port: 20331
+    TLSConfig:
+        Enabled: true
```

Comment:
By default? It'll fail here and there on every machine without keys.

Reply:
Fixed.